bundler/lib/bundler/cli/install.rb

```diff
@@ -53,7 +53,7 @@ def run
 
       if options["binstubs"]
         Bundler::SharedHelpers.major_deprecation 2,
-          "The --binstubs option will be removed in favor of `bundle binstubs`"
+          "The --binstubs option will be removed in favor of `bundle binstubs --all`"
```

Comment:
From what I understand if someone is looking for the behavior they had before using `bundle install --binstubs` which was to create binstubs for the whole bundle, `bundle binstubs --all` would be the command/ flag combination to get that. Is that a correct understanding?

Reply:
That's correct